bundles/org.openhab.binding.ws980wifi/README.md

```diff
@@ -0,0 +1,92 @@
+# ws980wifi Binding
```

Comment:
Here and all other places where binding name appears:
```suggestion
# WS980WiFi Binding
```

Unless there is any specific reason for the binding name being in lowercase when the product name is WS980WiFi?

Reply:
ws980wifi is the name of the binding.
WS980WiFi is the product name of ELV

Reply:
ws980wifi is the name of the binding, WS980WIFI is the name of the product w from ELV.

Reply:
My question is: Why is the name of the binding in lowercase?

Reply:
After looking through all files again and noticing binding name and thing type being the same, I'm wondering if you really want the model number to be part of the binding name? This would obsolete the entire binding in case another weather station model would have to be supported. Consider if the binding name could be more generic, for example something like "ELV Weather Station"? Only supported thing for now would then be "WS980WiFi", but at least more could be added.

Reply:
>  I'm wondering if you really want the model number to be part of the binding name? This would obsolete the entire binding in case another weather station model would have to be supported

This triggered me to look up what would be a good name and it seems to be even more generic. If what I see is correct, this weather station is a white-labelled device that is sold under many names. And there is another pr #12464 that supports the same stations. I think that binding might also work with the WS980WIFI weather station.

Reply:
@Hilbrand - that's a very good catch! This is a picture of Fine Offset WH2350:
![image](https://user-images.githubusercontent.com/19519842/168894429-11c48ae9-12fa-41b3-800d-5d5face3e55e.png)

And this is ELV WS980WiFi:
![image](https://user-images.githubusercontent.com/19519842/168894482-ac25995c-a12a-4591-864b-1a61864d8463.png)

As mentioned in #12464: "These weather stations are white labeled products which are re-branded by many distribution companies around the world."

I agree that indeed it looks like ELV WS980WiFi is indeed a re-branded Fine Offset weather station.

@prof2004 - notwithstanding the work you have put into this PR, can you check if the Fine Offset binding supports your weather station as well? @Andy2003, tagging you in case you have anything to add. Can you assess if anything additional would be needed in your binding to properly support this model as well, assuming that it's in fact a re-branded version of the same hardware and running with the same firmware?

Reply:
> @prof2004 - notwithstanding the work you have put into this PR, can you check if the Fine Offset binding supports your weather station as well?

After looking into this implementation it indeed looks like the same protocol is used as in the [fine-offset-weather-station-binding](https://github.com/openhab/openhab-addons/pull/12464). But this binding implements only a subset of [all the available measurands](https://github.com/Andy2003/openhab2-addons/blob/deae428938aa1829db1eb9b18b788c4b3f129e82/bundles/org.openhab.binding.fineoffsetweatherstation/src/main/java/org/openhab/binding/fineoffsetweatherstation/internal/domain/Measurand.java) is implemented.